mix/qml/js/InputValidator.js

```diff
@@ -0,0 +1,112 @@
+var nbRegEx = new RegExp('^[0-9]+$');
+function validate(model, values)
+{
+	var inError = [];
+	for (var k in model)
+	{
+		if (values[model[k].name])
+		{
+			var type = model[k].type.name;
+			var res;
+			if (type.indexOf("int") !== -1)
+				res = validateInt(type, values[model[k].name]);
+			else if (type.indexOf("bytes") !== -1)
+				res = validateBytes(type, values[model[k].name]);
+			else if (type.indexOf("bool") !== -1)
+				res = validateBool(type, values[model[k].name]);
+			else if (type.indexOf("address") !== -1)
+				res = validateAddress(type, values[model[k].name]);
+			else
+				res = validateAddress(type, values[model[k].name]); //we suppose that this is a ctr type.
```

Comment:
Better check for contract type explicitly so that this code would not fail if a new type is introduced. What is the type value in case of a contract? If it contains contract name this code would fail for contracts with names such as "MyboolContract"